services/director/src/simcore_service_director/producer.py

```diff
@@ -1020,12 +1020,27 @@ async def _save_service_state(service_host_name: str, session: aiohttp.ClientSes
         url=f"http://{service_host_name}/state",
         timeout=ServicesCommonSettings().director_dynamic_service_save_timeout,
     ) as response:
-        response.raise_for_status()
-        log.info(
-            "Service '%s' successfully saved its state: %s",
-            service_host_name,
-            f"{response}",
-        )
+        try:
+            response.raise_for_status()
+
+        except (web.HTTPMethodNotAllowed, web.HTTPNotFound) as err:
```

Comment:
I see, we do not care for services which do not expose a save endpoint.

Reply:
@GitHK As I write in the doc, i think i tis reasonable to assume that if a service does not implement a state entrypoint is because it does not ahve any feature.

Reply:
@GitHK @mguidon  you did not notice in the review that the exceptions i was using were not the right ones :-(
Developing in the director is difficult due to the legacy. It needs double/triple attention to review it.  We need to replace this service as soon as we can.